template/views/base/db/index.js

```diff
@@ -5,13 +5,14 @@ const mongoose = require("mongoose");
 // ℹ️ Sets the MongoDB URI for our app to have access to it.
 // If no env has been set, we dynamically set it to whatever the folder name was upon the creation of the app
 
-const MONGO_URI = process.env.MONGODB_URI || "mongodb://localhost/{{name}}";
+const MONGO_URI = process.env.MONGODB_URI || "mongodb://localhost:27017/{{name}}";
```

Comment:
interesting the port is necessary?

Reply:
The port 27017 is used for mongoDB server and the example reflects the values outlined in the Docs. The references for setting up the MongoDB connection string:
- https://www.mongodb.com/docs/drivers/node/current/fundamentals/connection/connect/
- https://mongoosejs.com/docs/connections.html